projects/1.16.1/assets/cazfps-the-dead-sea/cazfps_the_dead_sea/lang/zh_cn.json

```
@@ -0,0 +1,93 @@
+{
+  "advancements.tds.descr": "惊恐维度！",
+  "block.cazfps_the_dead_sea.steeltrapdoor": "钢活板门",
+  "block.cazfps_the_dead_sea.concretwall": "混凝土墙",
+  "item.cazfps_the_dead_sea.forestghost": "死森幽魂 刷怪蛋",
+  "item.cazfps_the_dead_sea.deadskeleton": "死亡骷髅 刷怪蛋",
+  "block.cazfps_the_dead_sea.steelwall": "钢墙",
+  "item.cazfps_the_dead_sea.deadclown": "死亡小丑 刷怪蛋",
+  "item.cazfps_the_dead_sea.oldman_spawn_egg": "可怖老人 刷怪蛋",
+  "block.cazfps_the_dead_sea.deadstonestairs": "死之石楼梯",
+  "advancements.tds.title": "死亡之海",
+  "entity.cazfps_the_dead_sea.deadwalker": "死亡游荡者",
+  "block.cazfps_the_dead_sea.deaddirt": "死亡土",
+  "item.cazfps_the_dead_sea.bigfoot_spawn_egg": "大脚巨人 刷怪蛋",
```

Bigfoot是西方市井传说中的怪物，在中文语境下长期被译为大脚怪。建议从俗。
```suggestion
  "item.cazfps_the_dead_sea.bigfoot_spawn_egg": "大脚怪 刷怪蛋",
```